python_modules/dagster/dagster/generate/new_project/new_project/repository.py.tmpl

```diff
@@ -13,8 +13,8 @@ def {{ repo_name }}():
     For hints on building your Dagster repository, see our documentation overview on Repositories:
     https://docs.dagster.io/overview/repositories-workspaces/repositories
     """
-    pipelines = [my_pipeline]
+    jobs = [my_graph.to_job()]
```

Comment:
It's strange that the job here is a different Python object instance than the job passed to the schedule / sensor.  If someone wanted to add resources, they would need to add them in multiple places.  It would be preferable to either:
* Define the job in the file where the graph is defined
* Avoid invoking `to_job` at all, and instead just directly pass the graph